corehq/tests/app_test_db_dependencies.yml

```diff
@@ -5,6 +5,19 @@
 #
 # This means that replacing INSTALLED_APPS with the list of declared dependencies plus the app
 # should still allow `./manage.py test app` to pass.
+
+# apps listed in the 'default' section will be included for all other apps
+default:
+  - django.contrib.auth
+  - django.contrib.contenttypes
+  - django.contrib.sessions
+  - two_factor
+  - django_otp.plugins.otp_static
+  - django_otp.plugins.otp_totp
+  - corehq.couchapps
+  - corehq.apps.tzmigration
+  - couchforms
+  - corehq.apps.users
```

Comment:
I ran some timings on a noop test and this is how much each package contributed. Def going to get rid of couchapps but I think at least tzmigration and couchforms can stay. Also keen to keep two_factor and django_otp since they are required by nearly 65% of the tests and only add  0.5s.
```
two_factor, django_otp	0.542
couchforms	                0.126
users                	        0.641
tzmigration	                0
couchapps	                4.631
```

Don't know of any other way to reduce the timings.